### DISCUSSION TITLE
Calculate Field's Edge - add Number to Map, save as Name

### DISCUSSION THREAD

**Discussion by realm174** [Courseplay/courseplay#7559] [Q&A] [answered]
Hello,
Using maps with no pre-defined fields (Waldsee, No Man's land, etc), I like to create my own fields and give them numbers.  Is there a way to have the numbers show up on the map? Or would that require manually modifying some image file? And if it is the case, can anyone point me in the direction of what I would need to modify to have the numbers show ?
thanks!

**Comment by Tensuko** [comment, marked as answer]
The Numbers on the Map are printed on it, not generated or something.
That is something for a mapper tutorial... not CP.

**Reply by realm174** [reply]
Well, thanks!  That was a simple question, not a request for tutorial.
